lambda_handler.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+''' Query Security Hub, export Inspector get_findings
+    boto reference: pagination and filtering: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html
+'''
+from botocore.vendored import requests
+import sys 
+import boto3
+import os
+import os.path
+import getpass 
+import configparser
+import base64 
+import base64
+#import rajworks
+from datetime import datetime, timedelta,date
+import datetime
+import logging
+
+import csv
+if sys.platform in ['win32', 'darwin']:
+    proxy_url = 'http://zsproxy.fanniemae.com:10479'
+else:
+    proxy_url = 'http://zsproxy.fanniemae.com:9480'
+
+region = 'us-east-1'
+outformat = 'json'
+today=date.today()
+print(today)
+filename='/tmp/InspectorFindings-' + str(date.today()) + '.csv'
+s3filename=filename.replace("/tmp/",'')
+
+print(filename)
+print(s3filename)
+
+
+region = 'us-east-1'
+outformat = 'json'
+sslverification = True
+awsconfigfile = '/.aws/credentials'
+
+
+
+##############################################################################################
+# Logging Configuration:
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
+# Comment/Uncomment the next line to enable/disable debug logging (to the screen)
+logging.disable(logging.CRITICAL)
+def lambda_handler(event, context):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This function contains 55 lines of code, not including blank lines or lines with only comments, Python punctuation characters, identifiers, or literals. By comparison, 99% of the functions in the CodeGuru reference dataset contain fewer lines of code. Large functions might be difficult to read and have logic that is hard to understand and test. We recommend that you simplify this function or break it into multiple functions.